sdk/storage/storage-queue/src/QueueClient.ts

```diff
@@ -1271,18 +1271,21 @@ export class QueueClient extends StorageClient {
   public async updateMessage(
     messageId: string,
     popReceipt: string,
-    message: string,
+    message?: string,
```

Comment:
Could we add what's the by design behavior when message is undefined into method's comments.

Reply:
Added comments "If this parameter is undefined, then the content of the message won't be updated."